src/pages/configurePatch/configurePatchCore/ConfigureBuildVariants.tsx

```diff
@@ -7,24 +7,26 @@ import { Body } from "@leafygreen-ui/typography";
 import { SiderCard } from "components/styles";
 import { Divider } from "components/styles/Divider";
 import { ProjectBuildVariant } from "gql/generated/types";
-import { VariantTasksState } from "pages/configurePatch/ConfigurePatchCore";
 import { array } from "utils";
+import { VariantTasksState } from "./state";
 
 const { toggleArray } = array;
 const { green } = uiColors;
 
 interface Props {
   variants: ProjectBuildVariant[];
   selectedVariantTasks: VariantTasksState;
-  selectedBuildVariant: string[];
-  setSelectedBuildVariant: React.Dispatch<React.SetStateAction<string[]>>;
+  selectedBuildVariants: string[];
```

Comment:
I noticed that selectedBuildVariants only is ever length 1 regardless of the number of build variants selected on the left panel. should this variable type `string` instead of `string[]`